src/__test__/myReservations.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import MyReservations from '../pages/MyReservations';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('MyReservations', () => {
+  it('should render correctly', () => {
+    const store = mockStore({
+      doctorsReducer: {
+        doctors: [
+          {
+            id: 7,
+            name: 'Peter Omaha',
+            speciality: 'General Practice',
+            description:
+              'Gradutated in the US, have six diplomas in medical genetic, also have a PhD in medicine and a master in medical genetics.',
+            graduation: '05/02/2010',
+            image:
+              'https://t4.ftcdn.net/jpg/03/05/41/27/360_F_305412791_XRNiWaFCREjLLpSQfj0e736foBoYXXYv.jpg',
+          },
+        ],
+      },
+      reservationsReducer: {
+        reservations: [
+          {
+            id: 8,
+            date: '2023-03-03',
+            city: 'Kampala',
+            user_id: 9,
+            doctor_id: 7,
+          },
+        ],
+      },
+    });
+
+    const { getByText } = render(
+      <Provider store={store}>
+        <MyReservations />
+      </Provider>,
+    );
+
+    // Check that the component renders the expected information
+    expect(getByText(/My appointments/i)).toBeInTheDocument();
+    expect(getByText(/Peter Omaha/i)).toBeInTheDocument();
+    expect(getByText(/General Practice/i)).toBeInTheDocument();
+    expect(getByText(/Friday, 3 March 2023/i)).toBeInTheDocument();
```

Comment:
when I run the test, appear me this error, is because not rendering the date correctly, could  you please fix this issue

<img width="899" alt="image" src="https://user-images.githubusercontent.com/107327344/222724936-3c2cbf10-c676-4d9c-8904-38630056f6f7.png">


<img width="754" alt="image" src="https://user-images.githubusercontent.com/107327344/222724999-0d71a6fa-7544-4068-a1f8-754bc9591136.png">